src/inspect_ai/tool/_tools/_computer/_computer_common.py

```diff
@@ -0,0 +1,127 @@
+import asyncio
+import json
+import logging
+
+from pydantic import BaseModel, Field
+
+from inspect_ai._util.content import ContentText
+from inspect_ai.model import Content, ContentImage
+from inspect_ai.tool._tool import ToolError
+from inspect_ai.tool._tools._computer._mock_logger import MockLogger
+from inspect_ai.util import sandbox
+
+# log = logging.getLogger(__name__)
+log = MockLogger()
+log.setLevel(logging.DEBUG)
+
+
+# TODO: Export ToolResult from inspect_ai.tool
+ToolResult = str | int | float | bool | list[Content]
+
+
+class ShellExecSuccessResult(BaseModel):
+    output: str | None = Field(default=None)
+    base64_image: str | None = Field(default=None)
+
+
+hackIsFirstCommand = True
+
+
+async def _send_cmd(cmdTail: list[str]) -> ToolResult:
+    # TODO: Resolve this issue
+    # without this delay, the first attempt to take a screenshot
+    # happens too soon before the GUI has actually rendered.
+    global hackIsFirstCommand
+    if hackIsFirstCommand:
+        stallResult = await sandbox().exec(["ls"])
+        if not stallResult.success:
+            log.error(f"First sandbox().exec() failed with: {stallResult.stderr}")
+            raise ToolError(f"Error executing command: {stallResult.stderr}")
+        log.debug("First sandbox().exec() succeeded...sleeping")
+        await asyncio.sleep(20)
+        log.debug("Stall done")
+        hackIsFirstCommand = False
+
+    cmd = ["python", "computer_tool_support/cli.py", "--action"] + cmdTail
```

Comment:
I noticed that `web_browser` copies itself to `/app/web_browser` -- I do think an absolute path is better here but I'm not sure about `/app`. Maybe we use `/opt/computer_tool` ?